Enigmatry.Entry.Blueprint.Api/ProgramExtensions.cs

```diff
@@ -106,7 +106,8 @@ public static void AppConfigureWebApplication(this WebApplication app)
         app.UseAuthorization();
 
         app.UseMiddleware<LogContextMiddleware>();
-
+        app.UseEntryExceptionHandler();
+        
```

Comment:
I would add a comment explaining why two UseEntryExceptionHandlers are addded. Otherwise I would be tempted to delete it :)